tensorflow/core/kernels/stack.cc

```diff
@@ -61,7 +74,10 @@ class Stack : public ResourceBase {
       return errors::InvalidArgument("Stack[", stack_name_, "] overflowed ",
                                      "its max_size (", max_size_, ")");
     }
-    stack_.push_back(value);
+    stack_.push_back({value, absl::nullopt, absl::nullopt});
+    int index = stack_.size() - 1;
```

Comment:
Can we change the type from int to size_t for all the indices?

Reply:
Yes, I've changed the indices to `size_t`.